### ISSUE TITLE
Accuracy of statement on unlicensed multi-contributor repos?

### ISSUE THREAD

**Issue by dhimmel** [github/choosealicense.com#503]
Currently the [no license page states](https://github.com/github/choosealicense.com/blob/749f888114dc092e8b6b19cfca0eb492722ddaa1/no-license.md):

> When you make a creative work (which includes code), the work is under exclusive copyright by default. Unless you include a license that specifies otherwise, nobody else can use, copy, distribute, or modify your work without being at risk of take-downs, shake-downs, or litigation. **Once the work has other contributors (each a copyright holder), “nobody” starts including you.**

I'm not sure the bold sentence is correct. From my understanding, the multiple contributors will become joint authors. As the article [Copyright in Open Source Software](https://doi.org/10.5033/ifosslr.v2i1.30) explains:

> Each co-owner of a joint work “obtains an undivided ownership in the whole of the joint work, including any portion thereof.”21  In other words, each co-owner may use or license the work, without the consent of other co-owners, in any way she may wish.22  Co-owners of a copyright do, however, owe to each other a duty to account for any income derived from their use or license of the work.23

So as long as you are a joint author, another joint author cannot prevent you from distributing or non-exclusively licensing the work. However, I am not a lawyer and don't know whether joint authorship is jurisdiction dependent. Also I'm not sure whether all repositories qualify as a joint work rather than a set of independent works.

Tagging @blahah.

**Comment by mlinksva** [comment]
Hi, thanks for bringing this up!

IANAL either, but have wondered about joint authorship and why it seems to get next to no play in open source. It's occasionally mentioned (the article you cite is a good example), but nothing definitive ventured (e.g, the only unambiguous situations mentioned in that article are sole copyright, whether through assignment or in the case of collective works; a similar text is <http://www.rosenlaw.com/pdf-files/Rosen_Ch02.pdf>) and certainly nothing definitive worldwide. What I've gathered (happy to be wrong, especially on this topic, that would make the world more interesting and probably better):

(1) Joint copyright is not treated uniformly around the world, e.g, compare US and [UK](https://www.gov.uk/guidance/ownership-of-copyright-works#joint-authors).
(2) One can't count on a court agreeing that a work is one of joint authorship rather than composed of separable components or a series of derivative works, e.g., a [GPL violation case in Germany](https://www.law.kuleuven.be/jura/art/44n4/peeters.html) where that was tried and failed.
(3) AFAIK most of the serious writing about joint authorship and something like distributed collaboration is more on the points of exploring why some potential joint authors aren't recognized as such or contrasting joint authorship cases with the practices of collaborative communities, rather than exploring claims of joint authorship in collaborative communities. Examples (both by the same author, Shun-Ling Chen) of these that I've enjoyed are https://ssrn.com/abstract=2598458 and https://ssrn.com/abstract=1826403 ... but my overall takeaway from these and some of the earlier papers they cite is that based on what cases exist, one can't rely much on joint authorship, despite the apparent simplicity of U.S. copyright law on joint authorship.

So maybe, possibly, interestingly, one could attempt to claim to be a joint author of a project to which one had started (or contributed to) but to which there are other contributors with no license or other agreement among them, and thus be free to non-exclusively exploit the project ... I don't know that anyone does that, and I doubt they'd do so free of risk of take-downs, shake-downs, or litigation. If anyone wishes to do this, consult with your own lawyer before doing so! Thus the text on the no-license page.

I'll leave this open for awhile in case anyone has corrections (which I would love!) or wants to suggest improved wording for the no-license page.

**Comment by dhimmel** [comment]
@mlinksva thanks for your detailed thoughts.

On one hand, I think either of the following modification would be more accurate:

> Once the work has other contributors (each a copyright holder), "nobody” **may include** you

> Once the work has other contributors (each a copyright holder), "nobody” **could start including** you

On the other hand, I like the forcefulness of "nobody starts including you"... I'm just not sure it's universally accurate, although you've convinced me its at least possible in many jurisdictions and circumstances. And I agree the ambiguity and non-deterministic determination of joint authorship creates risk for "take-downs, shake-downs, or litigation". However, no matter what steps you take, there will always be at least some risk of an attack of the :trollface:.

In conclusion, I'm fine with or without a change. I'm happy we started this conversation, so it's here for anyone who wants to dig deeper.

BTW @mlinksva, I see you help maintain the [Open Definition](https://github.com/okfn/opendefinition). Thanks! It's a fantastic resource that I've been referencing a lot when encouraging individuals license their scientific datasets.

**Comment by blahah** [comment]
I agree with all of the above, and think that a change to use *less certain language* would be helpful. Copyright is already so widely misunderstood, to the great detriment of us all. choosealicense.com generally does a great job of cutting through that, but clarity shouldn't come at the expense of accuracy. If something is more complex than a single succinct sentence can accurately capture, I think it's reasonable to make that clear.

**Comment by mlinksva** [comment]
"Risk" is already uncertain. The previous sentence, for maximum accuracy, also should include various caveats. I want to keep the direct wording. Risk is accurate, adding may merely makes sentence mushier, and last thing I want is for someone to read and go on a futile quest to figure out how they may not be at risk for using code from others that they don't have a license to; this is simply a waste of time as there is no definitive negative answer (especially not based on theory of joint authorship), only an evaluation of more or less risk. Again, someone really interested in pursuing such an evaluation needs to talk to their own lawyer, and it's way beyond the remit of this project.
